README.md

```diff
@@ -19,65 +42,34 @@ Mautic Introduction
 
-*Also note that the source outside <a href="https://github.com/mautic/mautic/releases">a tagged release</a> should be considered "alpha" and may contain bugs, cause unexpected results, data corruption or loss, and is not recommended for use in a production environment. Use at your own risk.*
+*Also note that source code outside of a [tagged release](https://github.com/mautic/mautic/releases) should be considered ‘alpha’. It may contain bugs, cause unexpected results, data corruption or loss, and is not recommended for use in a production environment. Use at your own risk.*
 
-## Ready to Install from Source and/or Contribute?
-That's fantastic! 
+### How to Install Mautic
+You must already have [Composer v1](http://getcomposer.org/) available on your computer because this is a development release and you'll need Composer to download the vendor packages. Note that Composer v2 is not yet supported.
 
-If you want to contribute to Mautic's **code**, please read our [CONTRIBUTING.md](https://github.com/mautic/mautic/blob/feature/.github/CONTRIBUTING.md) or [Contributing Code](https://contribute.mautic.org/contributing-to-mautic/developer) docs. Then, check out the issues with the [L1 label](https://github.com/mautic/mautic/issues?q=is%3Aissue+is%3Aopen+label%3AL1) to get started quickly :rocket:
+Installing Mautic is a simple three-step process:
 
-If you want to contribute in **other areas** of Mautic, please read our general [Contributing](https://contribute.mautic.org/contributing-to-mautic) guide.
+1. [Download the repository zip](https://github.com/mautic/mautic/archive/master.zip) then extract the zip to your web root.
+2. Run the `composer install` command to install the required packages.
+3. Open your browser and complete the installation through the web installer.
 
-## FAQ and Contact Information
-Marketing automation has historically been a difficult tool to implement in a business. The Mautic community is a rich environment for you to learn from others and share your knowledge as well. Open source means more than open code. Open source is providing equality for all and a chance to improve. If you have questions then the Mautic community can help provide the answers.
+If you get stuck, check our our [general troubleshooting](https://docs.mautic.org/en/troubleshooting) page. Still no joy? Join the [Mautic community](https://www.mautic.org/community) for support and answers.
 
-**Ready to get started with the community?** You can get <a href="https://www.mautic.org/community/get-involved">more involved</a> on the <a href="https://www.mautic.org">Mautic</a> website. Or follow Mautic on social media just to stay current with what's happening!
+### User Docs
```

Comment:
Prefer if we use the full term rather than abbreviation in headings :) 

```suggestion
### User Documentation
```

Reply:
Ha - sure. That's what happens when technical writers try to get funky. :)